content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,66 @@
+# Internal link checking tool
+
+You can use the https://github.com/wjdp/htmltest tool to check for broken links on **english content**. This is a helpful when refactoring sections of content, moving pages around, or renaming files or page headers.
+
+## What is and isn't checked
+
+The link checker scans generated HTML files, **not** raw markdown. You can see the full configuration in the `.htmltest.yml` file in the root of `kubernetes/website`.
+
+The link checker scans the following:
+
+- All content generated from markdown in the `/content/en/docs` directory **excluding**:
+  - Generated API references
+- All internal links **excluding**:
+  - Empty hashes (`<a href="#">` or `[title](#)`) and empty hrefs (`<a href="">` or `[title]()`)
+  - Internal links to images and other media files
+
+The link checker **does not** scan the following:
+
+- Links included in the site's "chrome": top and side menu links, footer links, links in a page's `<head>` section such as those to CSS stylesheets, scripts, and meta information
+- Top level pages _and_ their children, for example: `/training`, `/community`, `/case-studies/adidas`
+- Blog posts
+- API Reference documentation
+- Localizations
+
+## Prerequisites and installation
+
+[Docker](https://docs.docker.com/get-docker/) installed locally.
+
+The tool runs using a `make` command, so that's it!
+
+## Running the link checker
+
+To run the link checker:
+
+1. Navigate to the root directory of your local `kubernetes/website` repository.
+
+2. Run the following command:
+
+  ```
+  make check-internal-links
+  ```
+
+## Understanding the output
+
+If the link checker finds broken links, the output is similar to the following:
+
+```
+tasks/access-kubernetes-api/custom-resources/index.html
+  hash does not exist --- tasks/access-kubernetes-api/custom-resources/index.html --> #preserving-unknown-fields
+  hash does not exist --- tasks/access-kubernetes-api/custom-resources/index.html --> #preserving-unknown-fields
+```
+
+This is one "set" of broken links. More output is added to the log for each page with broken links.
+
+In this output:
+
+- The file with broken links is `tasks/access-kubernetes-api/custom-resources.md`.
+- The reason is `hash does not exist`. In most cases, you can ignore this.
+- The URL linked to is `#preserving-unknown-fields`.
```

Comment:
Only make lists if you're enumerating things.

Handy vocabulary from my SEO past: "source URL" and "target URL" for origin and destination.
```suggestion
In this output, the file with broken links is `tasks/access-kubernetes-api/custom-resources.md`.

The tool gives a reason: `hash does not exist`. In most cases, you can ignore this.

The target URL is `#preserving-unknown-fields`.
```